htmd/projections/metriccoordinate.py

```diff
@@ -35,11 +35,25 @@ class MetricCoordinate(Projection):
     -------
     metr : MetricCoordinate object
     """
-    def __init__(self, refmol, atomsel, trajalnstr='protein and name CA', refalnstr=None, centerstr='protein', pbc=True):
```

Comment:
please make parameter name more legible.
I don't understand them, why atomsel is not atomselstr, why having str everywhere?
Do not shorten vocals (and in case you do g is not a vocal in align)
centerstr should refer in the name to wrapping
can pbc go away?

Reply:
Hm yes the old naming is not great, I agree. I didn't shorten vocals in the case of `aln` but shorten it phonetically because I think I've seen this used in other software but ok. I prefer not to have huge argument names so I tried to find a compromise. But this is definitely a taste thing on which noone will ever agree :) The `pbc` and `center` option is in many other classes the same way so I would prefer not to change it.